docs/json-api.md

```diff
@@ -352,7 +352,7 @@ Compiles to:
 ```javascript
 const styles = {
   button: {
-    color: ['red', '!important'],
+    color: [['red'], '!important'],
```

Comment:
oh I just realized, it depends on jss-expand, margin: [5, 10, '!important'] would work, but not color: ['red', '!important']

Reply:
Oh, so shouldn't we document this then?

Reply:
it is basically jss-expand documentation, maybe we should cut it a little bit in the json-api.md as well?